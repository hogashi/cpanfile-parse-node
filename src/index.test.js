const index = require('../dist/index');

it('parse cpanfile', () => {
  return expect(index.parseCpanfile()).resolves.toMatchSnapshot();
});
