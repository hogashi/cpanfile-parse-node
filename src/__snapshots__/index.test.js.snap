// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parse cpanfile 1`] = `
"requires 'CPAN::Meta', 2.12091;
requires 'CPAN::Meta::Prereqs', 2.12091;
requires 'parent';

recommends 'Pod::Usage';

on test => sub {
    requires 'Test::More', 0.88;
    requires 'File::pushd';
};
"
`;
